config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: quay.io/metal3-io/ipam:master
+      - image: quay.io/metal3-io/ipam:main
```

Comment:
I am a bit confused here, the main tag should already exist in quay.io if I am not wrong and how do we trigger image build with that tag?